android/app/src/main/java/org/fossasia/openevent/adapters/FeedAdapter.java

```diff
@@ -150,4 +158,18 @@ public void onBindViewHolder(RecyclerViewHolder holder, int position) {
     public interface AdapterCallback {
         void onMethodCallback(List<CommentItem> commentItems);
     }
+
+    public void setOnImageZoomListener(OnImageZoomListener onImageZoomListener) {
+        this.onImageZoomListener = onImageZoomListener;
+    }
+
+    public void removeOnImageZoomListener() {
+        if (onImageZoomListener != null) {
+            onImageZoomListener = null;
+        }
+    }
+
+    private void zoomImage(String imageUri) {
+        onImageZoomListener.onZoom(imageUri);
```

Comment:
Suppose that the ZoomListener was not null at the time of construction but it is removed later on. Now once the image is clicked, NPE would be thrown.